doc/manual-wiki/misc.wiki

```diff
@@ -28,8 +28,8 @@ The function <<a_api | value Eliom_content.Html.D.make_uri>>
 
 
 
-To simplify the creation of <<span class="code"|<link>>> tags
-      for CSS or <<span class="code"|<script>>> tags for Javascript,
+To simplify the creation of <<span class="teletype"|<link>>> tags
+      for CSS or <<span class="teletype"|<script>>> tags for Javascript,
```

Comment:
We seem to have a `code` class in our HTML/CSS, but I can't find any references to `teletype`. Have you seen it somewhere?

Reply:
My bad. I noticed that the `<span>` elements with the `code` class on [miscellaneous wiki page](http://ocsigen.org/eliom/6.2/manual/misc#basic_menu) are note being rendered as code. For example, see the paragraph

>Here, home, infos, and tutorial are your three pages (generated for example by Eliom_service.​App.​create). 

which is defined by this HTML:

```
<p xmlns="http://www.w3.org/1999/xhtml">Here, <span class="code">home</span>,  <span class="code">infos</span>,
and <span class="code">tutorial</span> are your three pages (generated for example
by  <span><a class="ocsforge_doclink_eliom caml_link caml_request_node" style="white-space: pre-line;" data-eliom-cookies-info="[0,0,[0,&quot;eliom&quot;,[0,&quot;6.2&quot;,[0,&quot;api&quot;,[0,&quot;client&quot;,[0,&quot;Eliom_service.App&quot;,0]]]]]]" href="../api/client/Eliom_service.App#VALcreate" data-eliom-node-id="server_AAANPYNFRy1DxACDOXJjiUlCSZaE2g41d66d9e772ca394">Eliom_service.​App.​create</a></span>).
</p>
```

On the other hand, on pages such as [Communication between the client and the server](http://ocsigen.org/eliom/6.2/manual/clientserver-communication), the inline code is displaying correctly, and the underlying HTML looks like, e.g., `<span xmlns="http://www.w3.org/1999/xhtml" class="teletype">'a -&gt; 'b Lwt</span>`. I then jumped to the conclusion that I could add the `teletype` class in this angle bracket markup, without actually consulting the markup generating these pages. Now that I've done my due diligence, I see that the markup being used for inline code is `{{{'a -> 'b Lwt}}}`. 

I'll make the appropriate changes. Thanks for catching this and I apologize for the oversight!